Resources/Private/TypoScript/Root.ts2

```diff
@@ -54,8 +52,10 @@ prototype(RobertLemke.Plugin.Blog:Post) < prototype(TYPO3.Neos:Content) {
 	numberOfComments = ${q(node).children('comments').children('[spam = false]').count()}
 
 	@cache {
+		mode = 'cached'
 		entryIdentifier {
 			node = ${node}
+			content = 'blogpost'
```

Comment:
As per one of the commit messages, it avoids using the same entry identifier for the Page and the Post TS objects. If there's just the node, both have the same identifier, breaking the flushing of the cache.